src/ui/ui_i18n/README.md

```diff
@@ -320,10 +305,13 @@ global object exists in the runtime. `Intl` is present in all modern
 browsers and Node.js 0.10+. In order to load i18n engine
 in Node.js we should simply `import` this module (in Node.js, the
 [data](https://github.com/yahoo/intl-messageformat/tree/master/dist/locale-data)
-for all 200+ languages is loaded along with the library):
+for all 200+ languages is loaded along with the library) and pass the translation
+messages into the constructor:
 
 ```js
-import i18n from 'kbn-i18n';
+import I18n from '@kbn/i18n/src/core';
```

Comment:
You can actually say `import I18n from @kbn/i18n;` because inside of `packages/kbn-i18n/package.json` you'll have a line `"main": "src/index.js"` (if you wanted to write all this in typescript and build that, you'd have something more like `"main": "target/index.js"`).